homeassistant/components/mqtt/climate.py

```diff
@@ -749,6 +749,7 @@ async def _set_temperature(
 
     async def async_set_temperature(self, **kwargs: Any) -> None:
         """Set new target temperatures."""
+        operation_mode: str | None
```

Comment:
I think this should be `HVACMode`
```suggestion
        operation_mode: HVACMode | None
```